README.md

```
@@ -1,194 +1,379 @@
+    )
+).show(supportFragmentManager, "updater")
+```
+
+---
+
+## DSL Builders
+
+Both Compose and Fragment APIs offer Kotlin DSL builders for a more expressive syntax.
 
-AppUpdaterDialog.getInstance(data).show(supportFragmentManager, "UPDATE_DIALOG")
+#### Fragment DSL
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)